db/migrate/20140714061328_create_delayed_jobs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateDelayedJobs < ActiveRecord::Migration
+  def self.up
+    create_table :delayed_jobs, :force => true do |table|
+      table.integer  :priority, :default => 0, :null => false # Allows some jobs to jump to the front of the queue
```

Comment:
Line is too long. [114/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.